examples/ecs/system_sets.rs

```diff
@@ -51,6 +51,11 @@ fn main() {
     App::build()
         .add_plugins(DefaultPlugins)
         .init_resource::<Done>()
+        // Note that the system sets added in this example sets their run criteria explicitly.
```

Comment:
nit: "Note that the system sets added in this example _set_ their run criteria explicitly."

Reply:
Fixed.